src/helpers/plainAddPlaceholder/getPagesDictionaryRef.js

```diff
@@ -3,15 +3,20 @@ import SignPdfError from '../../SignPdfError';
 /**
  * @param {Object} info As extracted from readRef()
  */
-export default function getPagesDictionaryRef(info) {
-    const pagesRefRegex = new RegExp('\\/Type\\s*\\/Catalog\\s*\\/Pages\\s+(\\d+\\s\\d+\\sR)', 'g');
-    const match = pagesRefRegex.exec(info.root);
-    if (match === null) {
+const getPagesDictionaryRef = (info) => {
+    const catalogPosition = info.root.toString().indexOf('/Type/Catalog');
```

Comment:
So, a dounf by @therpobinski, this peace brakes the function.

Reply:
Excellent, it is not only the regular expression but the way of handling the error. That would solve everything and cover all unit tests. I will make a new branch at the fork of @andres-blanco by uploading the solution and requesting a mix to the develop branch.
Thank you.